nixos/modules/services/networking/keepalived/default.nix

```diff
@@ -150,6 +150,14 @@ in
         '';
       };
 
+      openFirewall = mkOption {
+        type = types.bool;
+        default = true;
```

Comment:
`openFirewall` should default to `false`.

> For example, enabling a module should not open firewall ports by default.[^1]

[^1]: https://github.com/NixOS/nixpkgs/blob/1382337b866618a7914d36afbc5014fc66404574/nixos/README.md#new-modules

There are only very few exceptions to this.

Reply:
Fixed here bd83588